### DISCUSSION TITLE
Change the location of “export ” in job scripts

### DISCUSSION THREAD

**Discussion by Zhang-pchao** [deepmodeling/dpdispatcher#221] [General]
In more cases, should the location of "export" be placed in front of "source"?
For example, it applies to the following situations (otherwise my_package cannot be found):

export PATH="/home_path/anaconda/envs/my_package/bin:$PATH"
source activate my_package

**Comment by njzjz** [comment]
Indeed, you can use the absolute path:
```sh
source /home_path/anaconda/envs/my_package/bin/activate my_package
```
So, I don't think it's necessary.
